src/ripple/app/ledger/impl/InboundLedger.cpp

```diff
@@ -64,6 +63,9 @@ enum
     ,reqNodes = 8
 };
 
+// millisecond for each ledger timeout
+auto constexpr ledgerAcquireTimeoutMillis = 2500ms;
```

Comment:
`Millis` in the variable name is redundant now that it has a chrono type, and may become incorrect if we change the timeout in the future (eg. `4s`).